redis-pq-demo/requirements.txt

```diff
@@ -0,0 +1,4 @@
+aries-cloudagent
```

Comment:
do we need this file?

Reply:
no as we are using a base image that already has those components.